x-pack/plugins/security_solution/public/common/components/events_viewer/index.tsx

```diff
@@ -137,6 +142,10 @@ const StatefulEventsViewerComponent: React.FC<Props> = ({
     }
     return () => {
       deleteEventQuery({ id, inputId: 'global' });
+      if (editorActionsRef.current) {
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do you need this here? i thought just above the dependency array

Reply:
yes, It is needed here too, this error pops if not diabled:
```
The ref value 'fieldEditorActionsRef.current' will likely have changed by the time this effect cleanup function runs. If this ref points to a node rendered by React, copy 'fieldEditorActionsRef.current' to a variable inside the effect, and use that variable in the cleanup function
```